testsuite/src/it/java/org/graalvm/tests/integration/RuntimesSmokeTest.java

```diff
@@ -106,6 +108,10 @@ public void testRuntime(TestInfo testInfo, Apps app, Map<String, String> switchR
             Logs.appendln(report, appDir.getAbsolutePath());
             Logs.appendlnSection(report, String.join(" ", cmd));
 
+            if (app.runtimeContainer != ContainerNames.NONE) {
+                waitForContainerLogToMatch(app.runtimeContainer.name, Pattern.compile(".*started.*"), 5, 1, TimeUnit.SECONDS);
```

Comment:
That makes sense. Do you think lowering the timeout and sleep time would be problematic? Waiting 5 seconds to timeout seems quite long to me, as well as waiting a whole second to retry.